### DISCUSSION TITLE
IntelliJ IDEA 2023.1 Copilot Plugin disables cmd+z and cmd+shift+z shortcuts

### DISCUSSION THREAD

**Discussion by pasha-vuiko** [community/community#51320] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hello! 
After updating to IntelliJ IDEA Ultimate version 2023.1 on MacOS, I noticed that Copilot Plugin disables undo (cmd+z) and redo (cmd+shift+z) shortcuts. When the plugin is disabled everything works properly.

**Comment by benjaminrobinet** [comment]
Same here, making me crazy

**Reply by giovannimartelli** [reply]
Same here, already opened a bug to IntelliJ Team. 
https://youtrack.jetbrains.com/issue/IDEA-316602/Undo-Redo-disabled-randomly-working-on-MacOS

**Comment by pasha-vuiko** [comment]
Also I've noticed, that the IDE starts with error when the plugin is enabled

**Comment by pasha-vuiko** [comment]
Looks like it's fixed. Thank You!
